app/editor/src/features/storage/StorageListView.tsx

```diff
@@ -39,95 +17,11 @@ export const StorageListView: React.FC = (props) => {
 
   useTooltips();
 
-  React.useEffect(() => {
-    storage.getFolder(path).then((data) => {
-      setFolder(data);
-    });
-  }, [path, storage]);
-
-  React.useEffect(() => {
-    if (!!streamUrl && !!videoRef.current) {
-      videoRef.current.src = streamUrl;
-    }
-  }, [streamUrl, videoRef]);
-
-  const selectItem = (item?: IItemModel) => {
-    setItem(item);
-    setStreamUrl(
-      !!item ? `/api/editor/storage/stream?path=${folder.path}/${item.name}` : undefined,
-    );
-  };
-
-  const deleteItem = async (item: IItemModel) => {
-    try {
-      await storage.delete(`${folder.path}/${item.name}`);
-      selectItem();
-      setFolder({ ...folder, items: folder.items.filter((i) => i.name !== item.name) });
-      toast.success(`${item.name} has been deleted`);
-    } catch {
-      // Ignore error a toast would have already been displayed with the error.
-    }
-  };
-
-  const items = folder.items.map((i) => {
-    if (i.isDirectory)
-      return (
-        <li
-          key={i.name}
-          onClick={() => {
-            selectItem();
-            setPath(`${folder.path}/${i.name}`);
-          }}
-        >
-          <div>
-            <FaRegFolder />
-            <span className="navigate">{i.name}</span>
-          </div>
-        </li>
-      );
-    return (
-      <li key={i.name}>
-        <Row nowrap>
-          <Row flex="2 1 0">
-            {isNotVideoOrAudio(i.name) ? <FaRegImage /> : <FaPhotoVideo />}
-            <span className="file" onClick={() => setItem(i)}>
-              {i.name}
-            </span>
-          </Row>
-          <Row flex="1 1 0">
-            {!isNotVideoOrAudio(i.name) && (
-              <FaPlay
-                className="stream"
-                data-for="main-tooltip"
-                data-tip="watch/listen"
-                onClick={() => selectItem(i)}
-              />
-            )}
-            <FaCloudDownloadAlt
-              className="download"
-              data-for="main-tooltip"
-              data-tip="download"
-              onClick={() => storage.download(`${folder.path}/${i.name}`)}
-            />
-            <FaTrash
-              className="delete"
-              data-for="main-tooltip"
-              data-tip="delete"
-              onClick={() => deleteItem(i)}
-            />
-          </Row>
-        </Row>
-      </li>
-    );
-  });
-
   return (
     <styled.StorageListView>
-      <Show visible={!!items.length}>
-        <FormikForm onSubmit={() => {}} initialValues={form}>
-          <ContentClipForm content={form} setContent={setForm} setClipErrors={setClipErrors} />
-        </FormikForm>
-      </Show>
+      <FormikForm onSubmit={() => {}} initialValues={form}>
+        <ContentClipForm content={form} setContent={setForm} setClipErrors={setClipErrors} />
```

Comment:
This is not a good implementation.  I wish I had caught this when it was done.  The `<ContentClipForm/>` should not be the shared generic component.  I will need to find time to do this correctly down the road...  More tech debt